docs/implementing.html

```diff
@@ -236,7 +236,7 @@ <h2 id="static-header-footer">Static header and footer</h2>
 <tr>
 	<th>Special assets</th>
 	<td>All included in the current GCWeb release</td>
-	<td><a href="https://github.com/wet-boew/GCWeb/blob/master/site/pages/gcweb-theme/static-header-footer/site-assets/x-bootstrap-4.css" rel="external">Download experimental CSS</a></td>
+	<td><a href="https://github.com/wet-boew/GCWeb/docs/static-header-footer/site-assets/x-bootstrap-4.css" rel="external">Download experimental CSS</a></td>
```

Comment:
This is still a broken link, I suggest using the following url: https://github.com/wet-boew/GCWeb/blob/master/docs/static-header-footer/site-assets/x-bootstrap-4.css

Reply:
The broken link at line 239 has been fixed